ElevatorSim/src/elevatorsim/elevator/Elevator.java

```diff
@@ -5,28 +5,48 @@
 import elevatorsim.enums.MessageDestination;
 import elevatorsim.scheduler.Scheduler;
 
+/**
+ * The elevator class
+ * Currently this is just used to receive messages from the scheduler
+ * and redirect them to the floors
```

Comment:
Add author? Also should we just put everyones names as the authors for all classes?

Reply:
added author. I put full name then student number in brackets.